zcash_client_sqlite/src/wallet.rs

```diff
@@ -2111,6 +2111,56 @@ pub(crate) fn get_received_memo(
         .transpose()
 }
 
+fn parse_tx<P: consensus::Parameters>(
```

Comment:
Nit: the diff for 1b78d1e2bfba0a3bd0cf51325a96583221885650 would be much cleaner if `parse_tx` were just below `get_transaction` instead of just above it.

Reply:
I know that it's old-fashioned, but my C habits of defining functions before they're used haven't completely gone away. :)